libs/wingcompiler/src/compile.ts

```diff
@@ -134,7 +134,7 @@ export function determineTargetFromPlatforms(platforms: string[]): string {
  */
 export async function compile(entrypoint: string, options: CompileOptions): Promise<string> {
   const { log } = options;
-  const preflightLog = options.preflightLog ?? console.log;
+  const preflightLog = options.preflightLog ?? process.stdout.write.bind(process.stdout);
```

Comment:
A test would be good to prevent this regression in the future

Reply:
Added